DETAILED_INSTALL.md

```diff
@@ -181,7 +181,34 @@ docker run --gpus all -it --rm  -v $(pwd):/home/app/mava -w /home/app/mava insta
         pip install id-mava[sc2]
         ```
 
-3. Run an example:
+    * MeltingPot:
+
+        Install MeltingPot:
+
+        ```bash
+        ./bash_scripts/install_meltingpot.sh
+        ```
+
+        Add MeltingPot to your python path:
+
+        ```bash
+        export PYTHONPATH="${PYTHONPATH}:${PWD}/../packages/meltingpot"
+        ```
+
+        If this fails, follow instructions [here](https://github.com/deepmind/meltingpot#installation).
+
```

Comment:
I think there was merging conflict if you can delete this section.

Reply:
Done :smile: